apis/backup.go

```diff
@@ -130,7 +134,10 @@ func (api *backupApi) download(c echo.Context) error {
 	if err != nil {
 		return NewBadRequestError("Failed to load backups filesystem.", err)
 	}
-	defer fsys.Close()
```

Comment:
If the editor does not give a warning for this type of use, it is not clear that an error message has not been handled.